server/routes/createGoal/createGoalController.test.ts

```diff
@@ -20,6 +20,9 @@ jest.mock('./addStepFormValidator')
 jest.mock('./createGoalFormValidator')
 jest.mock('./mappers/createGoalFormToCreateGoalDtoMapper')
 
+/**
+ * Unit tests for createGoalController.
+ */
```

Comment:
This is the original unit test class for the original `createGoalController` - the changes here are simply around the route paths, because thats the scope of the change that I've made in the implementation.